### PR TITLE
Alexandria Website Update

### PR DESCRIPTION
**Pull request by blrhc**
Updates to main page and new `/alexandria` page with countdown timer, links, assets updated etc.

### CODE REVIEW

**Review by bwhm — COMMENTED** [review]
I don't think I'm the right person to perform the review of neither "design" nor code, so will defer.

A few notes though:
- Overall, it looks pretty good to me!
- All links I tried are correct, and the info/maps/artwork etc. looks good
- Timer is wrong, as target is 15.09.20, 10:00 UTC
- Old testnets contain some incorrect data/stuff

My main question is what happens when timer reaches zero?
We can address this later, but AFAICT, what we would want
- Main page:
   - `Constantinople Testnet` gets dropped
   - `Alexandria Testnet` takes its place
   - Links change
- Constantinople page:
   - Networks replaced (similar to other Acropolis?)
   - See "Old testnets contain some incorrect data/stuff
- Alexandria page:
   - Network went live on...
   - `n` - run this role on Alexandria

**Comment by bwhm** [comment]
I think Francesco may not be needed for anything but review!

File: src/pages/alexandria/index.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      <SiteMetadata title="Joystream: The video platform DAO" description="Explore the Alexandria Testnet" />
+
+      <Hero
+        image={alexandriaImage}
+        title="Alexandria Network"
+        indent
+        chip={<Chip onClick={() => setModalClosed(true)}>What is this?</Chip>}
+        animationStartValue={0}
+      >
+        <p className="ConstantinoplePage__hero-paragraph">
```
Comment (Gamaranto) [inline src/pages/alexandria/index.js:49]: I think this is a typo and should be `AlexandriaPage__hero-paragraph`

File: src/pages/alexandria/style.scss [hunk 2]
```diff
@@ -0,0 +1,7 @@
+@import '../../styles/main.scss';
+
+.ConstantinoplePage {
```
Comment (Gamaranto) [inline src/pages/alexandria/style.scss:3]: Same here as above

**Review by Gamaranto — APPROVED** [review]

**Comment by Gamaranto** [comment]
looks good to me! 
I am only reviewing the technical aspect of the PR and making sure links, clock etc... are working, I leave comments on design and content to @bwhm or @bedeho

**Pull request merged**